book/src/kubernetes-changelog.md

```diff
@@ -3,6 +3,28 @@
 This page summarizes major CSI changes made in each Kubernetes release. For
 details on individual features, visit the [Features section](features.md).
 
+## Kubernetes 1.23
+* Beta
+    * Add Fsgroup Mount
```

Comment:
I would maybe call this one something like "Delegate fsgroup to CSI driver"

Reply:
updated